windows/src/developer/TIKE/compile/CompileKeymanWeb.pas

```diff
@@ -687,9 +795,20 @@ function TCompileKeymanWeb.JavaScript_OutputString(fkp: PFILE_KEY; pwszOutput: P
 
   pwsz := pwszOutput;
 
-  if Assigned(fkp)
-    then len := xstrlen_printing(fkp.dpContext)
-    else len := -1;
+  if Assigned(fkp) then
+    begin
```

Comment:
`begin` should be on the same indent as its parent `if` statement (and likewise `end`) - so we only get a single level of indent for the contained statements.